### DISCUSSION TITLE
How to install the R package ospsuite

### DISCUSSION THREAD

**Discussion by noxavier** [Open-Systems-Pharmacology/Forum#1391] [Questions & Problems] [answered]
I installed ospsuite-R according to the steps in the link（https://www.open-systems-pharmacology.org/OSPSuite-R/  ） ,but I encountered this problem when running ospsuite R. How can I solve it?
![捕获](https://user-images.githubusercontent.com/109507330/203746309-9d6af10c-6f5a-48a5-8b5f-e3d76e11e4dc.JPG)

**Comment by SRuedesh** [comment, marked as answer]
Hi @noxavier, 
the ospsuite-R package has some prerequisite packages you need to install before loading it. It seems you are missing at least two of those. Some of these packages require manual installs, such as `ospsuite.utils` (not on CRAN).
The documentation on the github page contains the instructions here: [https://github.com/Open-Systems-Pharmacology/OSPSuite-R#installation](https://github.com/Open-Systems-Pharmacology/OSPSuite-R#installation).
(Alternatively, if you have the `remotes` package installed, you should be able to install `ospsuite.utils` by running `remotes::install_github("Open-Systems-Pharmacology/OSPSuite.RUtils")` in your R console.)
As for `jsonlite`, you can install the package from CRAN by running `install.packages("jsonlite")` in your R console.
Hope this helps.
Best regards, Simeon

**Reply by noxavier** [reply]
Embarrassed! I didn't install ospsuite. utils. I thought it was the same as ospsuite. THANKS！
